app/utils/utils.js

```diff
@@ -54,13 +54,13 @@ const formatTimeIn = function (milliseconds, i18next = require('i18next')) {
 }
 
 // does not consider `postponesLimit`
-function canPostpone (postpone, passedPercent, postponePercent) {
-  return postpone && passedPercent <= postponePercent
+function canPostpone (postpone, passedPercent, postponePercent, hardcoreMode) {
```

Comment:
so in HC mode, I am not able to postpone or skip?  

`tests/utils.js` should be updated with new variable and test cases.